tests/testthat/test-load_files.R

```diff
@@ -195,7 +195,7 @@ test_that(".check_file_structure works as expected", {
     df[, !apply(df[seq_len(35), ], 2, function(x) {
       all(is.na(x))}), with = FALSE]
   Bckd_info_idx <-
-    which(as.data.frame(df)[, 1] %in% "Background information")
+    which(df[, 1] %in% "Background information")
```

Comment:
Why not?
```
    which(df[, "Plate information"] == "Background information")
```
We try to find the index of rows with value "Background information", right? 
And `%in%` seems not working as comparing column with single string.

Reply:
FYI,
```
> dt <- data.table(lt = sample(letters, size = 100,replace = TRUE))
> which(dt[["lt"]] == "a")
[1] 41 50 55 59 72
> which(dt[["lt"]] %in% "a")
[1] 41 50 55 59 72
> which(dt[["lt"]] %in% c("a", "b"))
[1]  4 41 50 55 59 72 84
> which(dt[, "lt"] == "a")
[1] 41 50 55 59 72
> which(dt[, "lt"] %in% "a")
integer(0)
> which(dt[, "lt"] %in% c("a", "b"))
integer(0)
```

I think we should switch to 
``` which(df[[ "Plate information"]] %in% "Background information")```
which will be the most universal solution working for any size of the character vector that follows "%in%". What do you think?